task_processing/plugins/mesos/mesos_executor.py

```diff
@@ -103,5 +101,8 @@ def stop(self):
         self.driver.stop()
         self.driver.join()
 
+    def status(self):
```

Comment:
This is just implementing an interface that we previously defined. We can discuss more on TASKPROC-30